emu/src/render/ppu.rs

```diff
@@ -61,42 +68,74 @@ impl PixelProcessUnit {
                 todo!("BG_MODE 2 not implemented yet")
             }
             3 => {
-                // Bitmap mode
+                acquire_lock!(self.internal_memory, memory => {
+                    // Bitmap mode
+                    for y in 0..LCD_HEIGHT {
+                        for x in 0..LCD_WIDTH {
+                            let index_color = (y * LCD_WIDTH + x) * 2;
+
+                            let color_array = [
+                                memory.video_ram[index_color],
+                                memory.video_ram[index_color + 1],
+                            ];
+
+                            let color: Color = [color_array[0], color_array[1]].into();
+
+                            acquire_lock!(self.gba_lcd, gba_lcd => { gba_lcd.set_pixel(x, y, color); })
+                        }
+                    }
+                });
+            }
+            4 => {
+                // 06000000-06009FFF for Frame 0
+                // 0600A000-06013FFF for Frame 1
```

Comment:
Why `06009FFF` and `06013FFF` instead of `060095FF` and `060135FF` as documentation says?

Reply:
Wrong copy paste. They are the GBC frames.